src/essence/Tools/Isochrone/IsochroneTool_Algorithm.js

```diff
@@ -0,0 +1,172 @@
+
+import Map_ from "../../Basics/Map_/Map_";
+const _X = 1, _Y = 0;
```

Comment:
why are these global?

Reply:
the algorithm stores pixel coordinates/array indexes as arrays of the form `[y, x]` (to match the order of indexing into a 2d array). these are the indexes of x and y values in those arrays. I thought it would be cleaner to be able to use x and y in those contexts, rather than having to think what 0 and 1 represent.

Reply:
sounds good. there's a convention where you prefix a variable with `_` to mean it's unused. I would just change this to `const X = 1, Y = 0;` to avoid confusion